src/pages/Profile/Profile.jsx

```diff
@@ -0,0 +1,73 @@
+import useTranslation from 'hooks/useTranslation';
+import { useLogoutMutation } from 'services/auth/auth';
+import useAuth from 'hooks/useAuth';
+import profile from 'assets/profile.svg';
+import smiles from 'assets/smiles.svg';
+import football from 'assets/football.svg';
+import travel from 'assets/travel.svg';
+import music from 'assets/music.svg';
+
+const Profile = () => {
+  const t = useTranslation();
```

Comment:
Maybe use a better name for this constant

Reply:
I agree, but this is a convention used by this RS template and I've seen this t variable in several projects. 
I made the same correction in other PRs but everyone disprove my comment.